controllers/datadogagent/override/podtemplatespec_test.go

```diff
@@ -443,3 +514,21 @@ func TestPodTemplateSpec(t *testing.T) {
 		})
 	}
 }
+
+func fakePodTemplateManagers(image string, t *testing.T) *fake.PodTemplateManagers {
```

Comment:
maybe you could rename that method to reflect that it's dedicated to give a podTemplateManager with images overidden?

Reply:
Renamed